packages/editor-ui/src/components/ParameterInputList.vue

```diff
@@ -171,11 +172,27 @@ export default mixins(
 
 				this.$emit('valueChanged', parameterData);
 			},
+
+			mustHideDuringCustomApiCall (parameter: INodeProperties, nodeValues: INodeParameters): boolean {
```

Comment:
I am sad we did not somehow bring all of this into node JSON instead...

Reply:
How do you mean? Adding a new `type` property to every `resource`, `operation` and `authentication` param in every node?

Reply:
well automating this at cli level instead of ui level

Reply:
Moved in this branch from UI to Server.ts